src/main/java/sandbox/SqlSandboxUtils.java

```diff
@@ -51,7 +51,7 @@ public static SQLiteConnectionPoolDataSource getDataSource(String databaseName,
             }
             Class.forName("org.sqlite.JDBC");
             SQLiteConnectionPoolDataSource dataSource = new SQLiteConnectionPoolDataSource();
-            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db");
+            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db?journal_mode=MEMORY");
```

Comment:
👍 though this seems like something that would belong in separate PR. it could potentially be confounding if something goes wrong when we merge this. we should also be more careful about handling SQL errors if this increases the number of them. maybe having a custom sql exception so we can handle and send an appropriate message to the user like "Please refresh the application" instead of  "[SQLITE_ERROR] SQL error or missing database (no such table: GLOBAL_RESOURCE_TABLE) "